source/_posts/2017/2017-08-14-how-to-apply-nette-coding-standard-in-your-project.md

```diff
@@ -0,0 +1,109 @@
+---
+id: 50
+layout: post
+title: "How to Apply Nette Coding Standard in Your Project"
+perex: '''
+    Coding standards help you to <strong>focus on architecture issues rather than code details in pull-requests</strong>, moreover in open-source. <strong>I wanted to make this possible in Nette</strong>, but Coding Standards could be found only in <a href="https://nette.org/en/coding-standard">documentation</a>.
+    <br><br>
+    This year I started to work on a Nette Coding Standard (*NCS*) you can put to CLI.
+    <br><br>
+    Today, you'll <strong>be able set it up in in your project, either open-source or closed</strong>. 
```

Comment:
I'm not native speaker, but sentence with "Today, you'll ..." sounds like czenglish ;-). What about to move the subject "Today" at the end of it?